r-package/grf/bindings/RegressionForestBindings.cpp

```diff
@@ -48,17 +48,23 @@ Rcpp::List regression_train(Rcpp::NumericMatrix train_matrix,
                             unsigned int samples_per_cluster,
                             bool compute_oob_predictions,
                             unsigned int num_threads,
-                            unsigned int seed) {
+                            unsigned int seed,
+                            bool mbb,
+                            unsigned int blocklength,
+                            unsigned int blocknum) {
   ForestTrainer trainer = regression_trainer();
 
   std::unique_ptr<Data> data = RcppUtilities::convert_data(train_matrix, sparse_train_matrix);
   data->set_outcome_index(outcome_index - 1);
-  if(use_sample_weights) {
```

Comment:
Why removing this?

Reply:
I thought you were asking me to remove these three lines. I will add it back.